addon/ng2/blueprints/ng2/files/src/test/matchers.js

```diff
@@ -0,0 +1,14 @@
+beforeEach(() => {
```

Comment:
we should move this into angular/angular/testing cli should not be in the business of distributing custom code that is not specifically needed for integration of various components.